### DISCUSSION TITLE
Welcome to ToDo_allpication Discussions!

### DISCUSSION THREAD

**Discussion by RobinKumar5986** [RobinKumar5986/ToDo_allpication#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

How the interface of todo application should be

**Comment by ghost** [comment]
Okay

**Reply by RobinKumar5986** [reply]
Tes
